pkg/apis/kops/v1alpha2/cluster.go

```diff
@@ -665,8 +665,10 @@ type EtcdManagerSpec struct {
 	// This allows etcd setting to be configured/overwriten. No config validation is done.
 	// A list of etcd config ENV vars can be found at https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/configuration.md
 	Env []EnvVar `json:"env,omitempty"`
-	// BackupInterval which is used for backups. The default is 15 minutes.
+	// BackupInterval which is used for backups.
 	BackupInterval *metav1.Duration `json:"backupInterval,omitempty"`
+	// BackupRetentionDays which is used for backups. The default is 90 days.
```

Comment:
Why is the default only mentioned in v1alpha2?

Reply:
Not sure how to put it. There is a new default for newly created clusters, otherwise the etcd-manager default, which varies.